wiki/Beatmap/Online_offset/id.md

```diff
@@ -10,10 +8,10 @@ tags:
 
 *Untuk penggunaan lainnya, lihat [Offset (disambiguasi)](/wiki/Disambiguation/Offset).*
 
-**Online offset** adalah sebuah offset yang diaplikasikan kepada beatmap setelah mendapat status ranked. Ini biasanya dilakukan jika sebuah beatmap membutuhkan sedikit perubahan timing untuk menyesuaikannya dengan musiknya. Nilai online offset ditentukan dari universal offset top player dan nilai lokal offset.
+**Online offset** adalah sebuah offset yang diaplikasikan kepada [beatmap](/wiki/Beatmap) setelah mendapat status ranked. Ini biasanya dilakukan jika sebuah beatmap membutuhkan sedikit perubahan timing untuk menyesuaikannya dengan musiknya. Nilai online offset ditentukan dari universal offset top player dan nilai offset lokal.
 
-Ini dapat diubah oleh anggota dari [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), dan beberapa pengguna seperti [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) dan [Developer](/wiki/People/The_Team/Global_Moderation_Team). Artinya, online offset tidak dapat dilihat oleh pengguna biasa pada situs web, namun dapat dilihat di dalam gim. Inilah tampilan yang diperlihatkan kepada pengguna yang diperbolehkan untuk melihat dan mengubahnya:
+Ini dapat diubah oleh anggota dari [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team), dan beberapa pengguna seperti [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) dan [Developer](/wiki/People/The_Team/Global_Moderation_Team). Artinya, online offset tidak dapat dilihat oleh pengguna biasa pada situs web, namun dapat dilihat di dalam permainan. Berikut adalah tampilan yang diperlihatkan kepada pengguna yang memiliki akses untuk melihat dan mengubahnya:
```

Comment:
GMT is developer?

Reply:
I will change that after I get clarification on which team member can edit online offset. I've asked BN, they are sure only NAT can change online offset.